index.html

```diff
@@ -11,22 +11,186 @@
+                            <img src="images/icon-languages.svg" alt="icon languages" class="icon">
+                            <h2 class="skills">Languages</h2>
+                            <ul class="buttons">
+                            
+                            <li><a class="button">JavaScript</a></li>
+                            <li><a class="button">Ruby</a></li>
+                            <li><a class="button">HTML</a></li>
+                            <li><a class="button">CSS</a></li>
+                        </ul>
+                        </div>
+                    </div>
+                    <div class="box2">
+                        <div class="skills">
+                            <img src="images/icon-frameworks.svg" alt="icon Frameworks" class="icon">
+                            <h2 class="skills">Frameworks</h2>
+                            
+                            <ul class="buttons">
+                            <li><a class="button">Bootstrap</a></li>
+                            <li><a class="button">Ruby on rails</a></li>
+                            <li><a class="button">RSpec</a></li>
+                            <li><a class="button">Capybara</a></li>
+                            <li><a class="button">Selenium</a></li>
+                        </ul>
+                        </div>
+                    </div>
+                    <div class="box3">
+                        <div class="skills">
+                            <img src="images/icon-skills.svg" alt="icon skills" class="icon">
+                            <h2 class="skills">Skills</h2>
+                            <ul class="buttons">
+                            <li><a class="button">Codekit</a></li>
+                            <li><a class="button">GitHub</a></li>
+                            <li><a class="button">Codepen</a></li>
+                            <li><a class="button">Gitlab</a></li>
+                            <li><a class="button">Terminal</a></li>
+                        </ul>
+                        </div>        
+                    </div>
+                </div>
+            <br>
+            <br>
+            <br>
+            <div class="aboutyou">
+                <div class="comment"><h2 class="aboutyou-title">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2></div>
+                <div class="form"><form action="https://formspree.io/f/mbjwboje" method="post">
+                    <input class="textarea" type="text" placeholder="name" maxlength="30" required>
+                    <input class="textarea" type="email" placeholder="email" required>
+                    <textarea class="textarea" name="mdr_message" id="message" cols="50" rows="10" maxlength="500" placeholder="Enter text here" required></textarea>
+                   <button class="touch" type="submit">Get in touch
+                        </button>
+                </form></div>
```

Comment:
- [ ] Please ensure that your items are properly aligned on
![Captura de pantalla 2022-06-10 173809](https://user-images.githubusercontent.com/59100477/173101715-f7b0f751-3fe9-4ea9-85b7-b6a01d06bcd9.jpg)
 the screen. Right now, your form is overlapping other components and the same with the button. Here's an image so you can see what I mean:

Reply:
- [ ] I fixed the contact section requested by previous reviewer. Please check below.

![image](https://user-images.githubusercontent.com/103516637/173124666-e23e1a0e-6875-44b0-8c50-7f89d769a169.png)